packages/web-components/src/components/va-textarea/va-textarea.tsx

```diff
@@ -131,7 +131,7 @@ export class VaTextarea {
       <Host>
         <label htmlFor="textarea">
           {label}
-          {required && <span class="required">(*{i18next.t('required')})</span>}
+          {required && <span class="required">{i18next.t('required')}</span>}
```

Comment:
These changes and the changes in the translation files are to make the new linting rule happy since it no longer sees string literals as JSX children.